doc/tutorials/make_your_first_quantum_circuit.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Since you have markdown cells above describing what happens, you can delete the comment from this cell :)</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>